src/containers/Dashboard/Tutorials/Tutorial.js

```diff
@@ -1,14 +1,54 @@
 import React, { Component, Fragment } from "react";
-import {
-  INTERVIEW_PREP_KIT,
-  STATISTICS,
-  JAVASCRIPT,
-  CODE
-} from "../../../constants";
+// import {
+//   INTERVIEW_PREP_KIT
+// } from "../../../constants";
 import "./Tutorial.scss";
 import TutorialCard from "./TutorialCard";
+import { addNotification } from "../../../utilities";
+import API from "../../../api/index";
 
 class Card extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      tutorials: [],
```

Comment:
@HarisShahidHameed why do we need two states here?

Reply:
it seems to be redundant.

Reply:
Removed extra state.